### PR TITLE
feat(cicd) fix build base image

### PR DESCRIPTION
**Pull request by bitsf**
fix some cicd build base image problems

Signed-off-by: Ziming Zhang <zziming@vmware.com>

### CODE REVIEW

File: tests/resources/Harbor-Util.robot [hunk 1]
```diff
@@ -93,7 +93,7 @@ Switch To LDAP
 Enable Notary Client
     ${rc}  ${output}=  Run And Return Rc And Output  rm -rf ~/.docker/
     Log  ${rc}
-    ${rc}  ${output}=  Run And Return Rc and Output  curl -o /notary_ca.crt -s -k -X GET --header 'Accept: application/json' -u 'admin:Harbor12345' 'https://${ip}/api/v2.0/systeminfo/getcert'
+    ${rc}  ${output}=  Run And Return Rc and Output  curl -o /notary_ca.crt -s -k -X GET --header 'Accept: application/json' -u 'admin:Harbor12345' 'https://${ip}/api/systeminfo/getcert'
```
Comment (reasonerjt) [inline tests/resources/Harbor-Util.robot]: I don't think the url is right.

File: tests/resources/Harbor-Util.robot [hunk 2]
```diff
@@ -52,21 +52,21 @@ Down Harbor
     Should Be Equal As Integers  ${rc}  0
 
 Package Harbor Offline
-    [Arguments]  ${with_notary}=true  ${with_clair}=true  ${with_migrator}=true  ${with_chartmuseum}=true  ${with_trivy}=true
+    [Arguments]  ${with_notary}=true  ${with_clair}=true  ${with_migrator}=true  ${with_chartmuseum}=true
```
Comment (reasonerjt) [inline tests/resources/Harbor-Util.robot]: why remove trivy?

File: tests/resources/Harbor-Util.robot (hunk 1 above)
Comment (wy65701436) [reply tests/resources/Harbor-Util.robot]: we should use `v2.0` in the path, otherwise, it should be 404

Comment (bitsf) [reply tests/resources/Harbor-Util.robot]: oh, my bad when fix conflict.

File: tests/resources/Harbor-Util.robot (hunk 2 above)
Comment (bitsf) [reply tests/resources/Harbor-Util.robot]: oh, my bad when fix conflict.

**Pull request merged**